lib/auth/accountrecovery.go

```diff
@@ -132,24 +110,10 @@ func (a *Server) verifyCodeWithRecoveryLock(ctx context.Context, username string
 		return nil
 	}
 
-	lockedUntil, maxedAttempts, err := a.recordFailedRecoveryAttempt(ctx, username)
-	switch {
-	case err != nil:
+	if err := a.recordFailedRecoveryAttempt(ctx, username); err != nil {
 		log.Error(trace.DebugReport(err))
```

Comment:
Should we add some context to this message, or does the underlying error carry everything we need?

Eg:

```suggestion
		log.WithError(err).Warn("Errored recording an account recovery failure")
```